indico/modules/events/surveys/forms.py

```diff
@@ -115,3 +121,22 @@ class ImportQuestionnaireForm(IndicoForm):
     json_file = FileField(_('File'), accepted_file_types="application/json,.json",
                           description=_("Choose a previously exported survey content to import. "
                                         "Existing sections will be preserved."))
+
+
+class EmailPeopleForm(IndicoForm):
```

Comment:
So I guess I should update that 
form too, with the recipients field and copy to sender option right ? Cause I didn't know about that form that we send emails there too.